src/Social/Models/ChannelMessages.php

```diff
@@ -17,4 +17,40 @@ public function initialize()
         
         $this->setSource('channel_messages');
     }
+
+    /**
+    * Return all the messages of a channel
+    *
+    * @param Channels $channel
+    * @param array $filter
+    *
+    * default $filter = ["page" => 1, "limit" => 25, "order_by" => "id", "sort" => "DESC"]
+    *
+    * @return Simple
+    */
+    public function getMessages(Channels $channel, int $page = 1, int $limit = 25, string $orderBy = "id", string $sort = "DESC"): Simple
```

Comment:
@rwhite27 you are going to hate me but I was refering to having sort by $sort = 'id|desc' , but this i good enough